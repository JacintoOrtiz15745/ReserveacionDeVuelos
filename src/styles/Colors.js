export const colors = {
  Purple: '#6472ee',
  ligthGray: '#b6b7ba',
  white: '#FFF',
  red: 'red',
  black: 'black',
  
};
