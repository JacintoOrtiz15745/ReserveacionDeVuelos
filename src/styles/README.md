This folder will contain styles if needed.