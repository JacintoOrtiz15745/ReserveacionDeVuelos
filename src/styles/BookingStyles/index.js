import {StyleSheet, Dimensions} from 'react-native';
import {colors} from '../Colors';
const {height} = Dimensions.get('window');

export const styles = StyleSheet.create({
  FlightTitle: {
    color: colors.black,
    fontSize: 35,
    fontWeight: 'bold',
    width: 300,
    marginBottom: 80,
  },
  BookingTextInput: {
    height: 70,
    borderBottomColor: colors.black,
    borderBottomWidth: 1,
    fontSize: 20,
    borderBottomColor: colors.ligthGray,
  },
  BookingButton: {
    backgroundColor: colors.Purple,
    padding: 12,
    borderRadius: 10,
    justifyContent: 'center',
  },
  BookingButtonText: {
    textAlign: 'center',
    color: colors.white,
    fontWeight: 'bold',
    fontSize: 18,
  },
  ScreenContainer: {
    margin: 20,
  },
  ContainerInformation: {
    flexDirection: 'column',
    justifyContent: 'space-between',
    height: '50%',
  },
  ContainerFlightInformation: {
    flexDirection: 'column',
    justifyContent: 'space-between',
    height: '50%',
  },
  ContainerForm: {
    marginTop: 50,
    flexDirection: 'column',
    justifyContent: 'space-between',
    height: '42%',
  },
  dateForm: {
    height: '80%',
    flexDirection: 'column',
    justifyContent: 'flex-start',
  },
  BookingInformation: {
    marginTop: 20,
    marginBottom: 5,
    flexDirection: 'row',
    justifyContent: 'space-between',
    borderBottomWidth: 0.5,
    borderColor: colors.ligthGray,
    height: 70,
  },
  InformationFlightsTitle: {
    fontSize: 25,
    fontWeight: 'bold',
  },
  InformationFlightsSubtitle: {
    color: colors.ligthGray,
    marginTop: 5,
  },
  left: {
    alignSelf: 'flex-start',
  },
  plane: {
    alignSelf: 'center',
  },
  BookingInf: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignItems: 'center',
    width: '55%',
  },
  item: {
    flexDirection: 'row',
    justifyContent: 'center',
    alignItems: 'center',
  },
  FlatList: {
    flexDirection: 'row',
    alignItems: 'center',
    flexDirection: 'row',
    justifyContent: 'center',
    alignContent: 'center',
    width: '50%',
    height: 50,
    alignSelf: 'center',
  },
  title: {
    fontSize: 33,
    fontWeight: 'bold',
  },
  Picker: {
    width: '100%',
    height: 50,
  },
  dateContainer: {
    margin: 20,
    flexDirection: 'column',
    justifyContent: 'flex-start',
    height: 525,
  },
  dateTextContainer: {
    flexDirection: 'column',
  },
  dateText: {
    borderColor: 'black',
    borderWidth: 1,
    marginTop: 10,
    width: 200,
  },
  bold: {
    fontWeight: 'bold',
  },
  doneContainer: {
    padding: 20,
    flexDirection: 'column',
    justifyContent: 'space-between',
    height: '100%',
  },
  doneInfo: {
    paddingTop: height * 0.2,
  },
});
