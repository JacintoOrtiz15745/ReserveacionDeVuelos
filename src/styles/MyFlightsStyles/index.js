import {StyleSheet} from 'react-native';
import {colors} from '../Colors';

export const styles = StyleSheet.create({
  myFlightsTitle: {
    color: colors.Purple,
    fontWeight: 'bold',
    fontSize: 35,
  },
  titleContainer: {
    margin: 10,
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignItems: 'center',
  },
  ListContainer: {
    padding: 15,
    height: 520,
  },
  TopContainer: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignItems: 'center',
  },
  middleContainer: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignItems: 'center',
    borderBottomWidth: 2,
    borderBottomColor: colors.ligthGray,
    marginTop: 5,
    marginBottom: 10,
  },
  bottomContainer: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignItems: 'center',
    marginBottom: 10,
  },
  fatherContainer: {
    marginBottom: 15,
    borderBottomWidth: 2,
  },
  country: {
    fontSize: 30,
    fontWeight: 'bold',
  },
  city: {
    color: colors.greyTitle,
  },
  border: {
    borderBottomColor: 'black',
    borderBottomWidth: 1,
  },
  dateTitle: {
    fontWeight: 'bold',
  },
  passengersTitle: {
    fontWeight: 'bold',
  },
  circleButton: {
    width: 60,
    height: 60,
    borderRadius: 60 / 2,

    alignSelf: 'center',
    backgroundColor: colors.Purple,
    flexDirection: 'row',
    justifyContent: 'center',
    alignItems: 'center',
  },
  screenContainer: {
    flexDirection: 'column',
  },
  BookingButton: {
    backgroundColor: colors.Purple,
    padding: 12,
    borderRadius: 10,
    justifyContent: 'center',
  },
  noFlights:{
    fontSize: 25,
    textAlign: 'center',
    color: colors.Purple
  }
});
