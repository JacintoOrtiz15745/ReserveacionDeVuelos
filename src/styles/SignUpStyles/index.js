import {colors} from '../Colors';
import {StyleSheet} from 'react-native';

export const styles = StyleSheet.create({
  title: {
    fontSize: 20,
    color: colors.Purple,
    fontWeight: 'bold',
    marginBottom: 10,
  },
  button: {
    height: 40,
  },
  SignUpContainer: {
    margin: 10,
    padding: 10,
  },
  textInput: {
    height: 40,
    borderColor: 'gray',
    borderWidth: 1,
    marginBottom: 10,
  },
  formText: {
    marginBottom: 10,
    fontSize: 16,
  },
  checkBoxContainer: {
    flexDirection: 'row',
    alignItems: 'center',
    justifyContent: 'flex-start',
    marginTop: 5,
  },
  btnSignupContainer: {
    marginTop: 15,
  },
  btnSignup: {
    backgroundColor: colors.Purple,
    padding: 12,
    borderRadius: 10,
    marginBottom: 5,
    flexDirection: 'row',
    justifyContent: 'center',
  },
  btnSignupText: {
    textAlign: 'center',
    color: colors.white,
    fontWeight: 'bold',
    fontSize: 18,
  },
  or: {
    textAlign: 'center',
    marginBottom: 10,
    marginTop: 10,
    color: colors.ligthGray,
  },
  LogInText: {
    color: colors.Purple,
  },
  loginTextContainer: {
    textAlign: 'center',
    marginTop: 10,
    flexDirection: 'row',
    justifyContent: 'center',
  },
  googleIcon: {
    width: 20,
    height: 20,
    marginTop: 3,
  },
  characters: {
    marginBottom: 20,
  },
  checkboxFather: {
    marginBottom: 15,
  },
  errorMessage: {
    color: colors.red,
  },
  signin: {
    margin: 10,
    padding: 10,
  },
  signinContainer:{
    marginBottom: 20,
    marginTop: 20,
    flexDirection: 'column',
    justifyContent: 'flex-start',
    height: 450
  }
});
