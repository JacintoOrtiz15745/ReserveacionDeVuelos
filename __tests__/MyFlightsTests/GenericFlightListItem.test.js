import React from 'react';
import {shallow} from 'enzyme';
import GenericListItem from '../../src/components/atoms/MyFlightsAtoms/GenericFlightListItem';
import {styles} from '../../src/styles/MyFlightsStyles';

describe('Renders correctly when there is no icon', () => {
  const props = {
    ContainerStyle: styles.topContainer,
    Title: 'title',
    Subtitle: 'subtitle',
    TitleFont: styles.country,
    SubtitleFont: styles.country,
    Icon: null,
  };
  const wrapper = shallow(<GenericListItem {...props} />);

  test('snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
  test('Render correctly', () => {
    const render = wrapper.findWhere((node) => node.prop('testID') === 'view');
    expect(render.exists()).toBeTruthy();
  });
  test('When there is no icon', () => {
    const icon = wrapper.findWhere((node) => node.prop('testID') === 'icon');
    expect(icon.exists()).not.toBeTruthy();
  });
});

describe('Icon When there is icon', () => {
  const props = {
    ContainerStyle: styles.topContainer,
    Title: 'title',
    Subtitle: 'subtitle',
    TitleFont: styles.country,
    SubtitleFont: styles.country,
    Icon: 'plane',
  };
  const wrapper = shallow(<GenericListItem {...props} />);
  test('When there is icon', () => {
    const icon = wrapper.findWhere((node) => node.prop('testID') === 'icon');
    expect(icon.exists()).toBeTruthy();
  });
  
});
