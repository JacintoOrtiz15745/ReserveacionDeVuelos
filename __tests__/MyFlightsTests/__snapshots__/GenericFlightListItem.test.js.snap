// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly when there is no icon snapshot 1`] = `
<View
  testID="view"
>
  <Text
    style={
      Object {
        "fontSize": 30,
        "fontWeight": "bold",
      }
    }
  >
    title
  </Text>
  <Text
    style={
      Object {
        "fontSize": 30,
        "fontWeight": "bold",
      }
    }
  >
    subtitle
  </Text>
</View>
`;
